install.sh

```diff
@@ -0,0 +1,117 @@
+export DEBIAN_FRONTEND=noninteractive
+
+echo '>> Updating information about packages...'
+sudo sh -c 'echo "deb http://cran.rstudio.com/bin/linux/ubuntu trusty/" >> /etc/apt/sources.list' # R
+gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9
+gpg -a --export E084DAB9 | sudo apt-key add -
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ wily-pgdg main" >> /etc/apt/sources.list.d/pgdg.list' # PostgreSQL
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo apt-get -y update
+sudo apt-get -y install vim
+
+echo '>> Installing xfce4 and VirtualBox guest tools...'
+sudo apt-get -y install xfce4 virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
+sudo sh -c 'echo "allowed_users=anybody" > /etc/X11/Xwrapper.config'
+
+echo '>> Installing pip...'
+wget --quiet https://bootstrap.pypa.io/get-pip.py
+sudo apt-get -y install python
+sudo python get-pip.py
+sudo apt-get -y install python-dev build-essential
```

Comment:
pip: either way is fine for me.
virtualenv: I thought the regular way to install it was using pip (as I do).